package.json

```diff
@@ -5,8 +5,23 @@
   "repository": "https://github.com/riskycase/my-express",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "pretest": "eslint --ignore-path .gitignore .",
+	"prettify": "prettier --single-quote --trailing-comma none --print-width 80 --write --no-bracket-spacing **/*.js"
```

Comment:
Prettier doesn't explicitly support es6 so --trailing-comma is set to none instead of es5